commands/audit/jas/applicability/applicabilitymanager_test.go

```diff
@@ -334,3 +334,25 @@ func TestParseResults_AllCvesNotApplicable(t *testing.T) {
 		assert.NotEmpty(t, applicabilityManager.applicabilityScanResults[0].Results)
 	}
 }
+
+func TestParseResults_ApplicableCveExist1(t *testing.T) {
```

Comment:
```suggestion
func TestParseResults_NewApplicabilityStatuses(t *testing.T) {
```

In addition I would consider merging the following tests, into testCases inside one method, to prevent duplicate code:
* This test method
* TestParseResults_EmptyResults_AllCvesShouldGetUnknown
* TestParseResults_ApplicableCveExist
* TestParseResults_AllCvesNotApplicable